### PR TITLE
update dvclive hf integration

### PR DESCRIPTION
**Pull request by dberenbaum**
Adds docs for dvclive callbacks within the huggingface libraries [`transformers`](https://github.com/huggingface/transformers/pull/27352) and [`accelerate`](https://github.com/huggingface/accelerate/pull/2139).

### CODE REVIEW

**Comment by dberenbaum** [comment]
This is ready to merge now. See https://github.com/iterative/dvclive/pull/740.

**Comment by dberenbaum** [comment]
ping @shcheklein

**Review by shcheklein — APPROVED** [review]

File: content/docs/dvclive/ml-frameworks/huggingface.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
-# Hugging Face
+# Hugging Face Transformers
 
 DVCLive allows you to add experiment tracking capabilities to your
```
Comment (shcheklein) [inline content/docs/dvclive/ml-frameworks/huggingface.md:3]: minor: add a reference here to accelerate. Can be a second sentence - "If you are using Accelerate ..."

**Comment by shcheklein** [comment]
Cool stuff, @dberenbaum . 

Good to have this better integration with HF.  Do we need to integrate with some other frameworks? Do we need to update / contribute to HF docs? Not related to this PR - It seems Catalyst could be dropped?

**Comment by dberenbaum** [comment]
> Good to have this better integration with HF. Do we need to integrate with some other frameworks? Do we need to update / contribute to HF docs? Not related to this PR - It seems Catalyst could be dropped?

We have pretty deep integration now with HF and Lightning:
- For HF, we are already in their docs for both [transformers](https://huggingface.co/docs/transformers/v4.36.1/en/main_classes/callback#transformers.integrations.DVCLiveCallback) and [accelerate](https://huggingface.co/docs/accelerate/usage_guides/tracking). 
- For lightning, they won't accept new 1st-party loggers, but adding the [fabric](https://dvc.org/doc/dvclive/ml-frameworks/fabric) integration puts us ahead of other trackers. [wandb](https://github.com/Lightning-AI/pytorch-lightning/issues/17885) support is not yet available, and mlflow and others haven't even started AFAICT.

Those are the [most important](https://plausible.io/dvc.org?page=%7E%2Fdoc%2Fdvclive%2Fml-frameworks&period=custom&keybindHint=L&from=2023-10-01&to=2023-12-31) these days IME. We have a [bunch more](https://github.com/iterative/dvclive/issues?q=is%3Aopen+is%3Aissue+label%3Ap1-important+label%3A%22A%3A+frameworks%22) to add or update, but I would not prioritize them until we have more resources.

**Pull request merged**